src/display/text_layer.js

```diff
@@ -226,7 +529,54 @@ var renderTextLayer = (function renderTextLayerClosure() {
           self._renderTimer = null;
         }, timeout);
       }
-    }
+    },
+    _expandTextDivs: function TextLayer_expandTextDivs(expand) {
+      if (expand) {
+        for (var i = 0, ii = this._textDivs.length - 1; i < ii; i++) {
```

Comment:
I'm skipping the last div because it's an end of content div. I don't think it has any of the actual text.